source/tyr/tyr/resources.py

```diff
@@ -943,3 +955,108 @@ def delete(self, billing_plan_id=None):
             logging.exception("fail")
             raise
         return ({}, 204)
+
+class Autocomplete(flask_restful.Resource):
+    def get(self, shape=None):
+        if shape:
```

Comment:
we want to filter on shape ?

maybe later but isn't the first use case a filter by Id (and name ?)